### PR TITLE
magento/graphql-ce#808: [Test coverage] Add disabled variation of Configurable Product to cart

### PR DESCRIPTION
**Pull request by sergiy-v**
### Description (*)
Covered the following test cases:
```
Magento\GraphQl\ConfigurableProduct\AddConfigurableProductToCartTest
- testAddDisabledVariationToCart
- testOutOfStockVariationToCart
```

### Fixed Issues (if relevant)

1. #808 

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: dev/tests/integration/testsuite/Magento/ConfigurableProduct/_files/product_configurable_disable_first_child.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
```
Comment (lenaorobei) [inline dev/tests/integration/testsuite/Magento/ConfigurableProduct/_files/product_configurable_disable_first_child.php:1]: Let's include `configurable` fixture here and disable child by `sku`.

File: dev/tests/integration/testsuite/Magento/ConfigurableProduct/_files/product_configurable_zero_qty_first_child.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?php
```
Comment (lenaorobei) [inline dev/tests/integration/testsuite/Magento/ConfigurableProduct/_files/product_configurable_zero_qty_first_child.php:1]: Let's include `configurable` fixture here and set zero quantity for the child by `sku`.

**Review by lenaorobei — CHANGES_REQUESTED** [review]
Please address review comments and add `rollback` files.

**Comment by sergiy-v** [comment]
> Please address review comments and add `rollback` files.

The mentioned changes has been added, please check the result.
Thank you.

**Comment by lenaorobei** [comment]
@magento run WebAPI Tests

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-6422](https://jira.corp.magento.com/browse/ENGCOM-6422) has been created to process this Pull Request
:eight_spoked_asterisk: @lenaorobei, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Pull request merged**